### PR TITLE
Support INCLUDE tag options like :lines, src, example ...

### PR DESCRIPTION
**Pull request by pierre-lecocq**
Now support these kind of INCLUDE tag options:

capture decran 2013-09-17 a 11 43 39

### CODE REVIEW

File: lib/org-ruby/parser.rb [hunk 1]
```diff
@@ -163,6 +163,49 @@ def parse_lines(lines, offset)
...
+        File.open(line.include_file_path, "r") do |fd|
+          while line_data = fd.gets
+            if (line_index >= include_file_lines[0] and (include_file_lines[1].nil? or line_index < include_file_lines[1]))
+              include_data << line_data.chomp
+            end
+            line_index += 1
+          end
+        end
+
+      when ('src' or 'example' or 'quote') then
```
Comment (wallyqs) [inline lib/org-ruby/parser.rb:190]: This will always eval to 'src', I think you mean `when 'src, 'example', 'quote'`

**Comment by wallyqs** [comment]
Hey, sorry for the late feedback. I tested using the following https://gist.github.com/wallyqs/6743698 and it seems ok (besides one trivial bug). Thanks for the contribution!

**Comment by pierre-lecocq** [comment]
thanks for the merge !

**Comment by wallyqs** [comment]
Hey @pierre-lecocq , I had another thought about this and decided that I will disable the feature by default.
Reason behind this is that it would be a bit dangerous in some environments if activated. Some sites (like Github for example) use this gem to allow users to render Org mode input into HTML, so if activated something like this would be totally possible:

``` org
* Dangerous example

#+INCLUDE: "/etc/passwd"
```

Instead I will make it possible to enable the feature with an environment variable like. So for example if you are using Jekyll, before exporting the site you would need to set `export ORG_RUBY_ENABLE_INCLUDE_FILES=true`.

**Comment by pierre-lecocq** [comment]
Hi @wallyqs !

I understand perfectly your opinion and agree with you.
Nevertheless, this feature is very useful.
Your solution (ENV var) is great, and maybe we could add another optional variable for "chrooting" includes.

Ex: export ORG_RUBY_INCLUDE_ROOT=/home/org/includes

If this var is set, the original ORG_RUBY_ENABLE_INCLUDE_FILES var is automatically set to true.

It could bring more security and more flexibility to the gem.

What do you think about that ?

Thanks for everything, have a great day.

P.

**Comment by wallyqs** [comment]
Hi @pierre-lecocq . I agree that having something like `ORG_RUBY_INCLUDE_ROOT` is a valid use case too. For now, I have added support for the `ORG_RUBY_ENABLE_INCLUDE_FILES`, and now I'm planning to include these both features in the upcoming release from the gem. 
Thanks for the feedback. Cheers!

**Comment by pierre-lecocq** [comment]
If you want me to do more on this part (or another one) in the future, do not hesitate to tell me !

**Comment by wallyqs** [comment]
@pierre-lecocq Of course, pull requests are always welcome! If you develop `ORG_RUBY_INCLUDE_ROOT` support I would be glad to merge it.

**Comment by pierre-lecocq** [comment]
Fine, I'll manage this ENV var during the coming week.

Thanks for everything @wallyqs !

**Comment by wallyqs** [comment]
@pierre-lecocq Great, thanks!

**Pull request merged**